.github/workflows/example-repository-dispatch.yml

```
@@ -0,0 +1,180 @@
+            echo "should_dispatch=true" >> $GITHUB_OUTPUT
+            echo "event_type=integration_test" >> $GITHUB_OUTPUT
+          else
+            echo "should_dispatch=false" >> $GITHUB_OUTPUT
+            echo "event_type=" >> $GITHUB_OUTPUT
+          fi
+
+  dispatch:
+    needs: build
+    runs-on: ubuntu-latest
+    if: needs.build.outputs.should_dispatch == 'true'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)